sandbox/version-control-manager/src/main/java/com/vikas/method_parser/version_control_manager/details/fetcher/UrlDetailesFetcher.java

```diff
@@ -0,0 +1,60 @@
+package com.vikas.method_parser.version_control_manager.details.fetcher;
```

Comment:
Why is this class named the way it is? The responsibility of this class should be to make REST calls irrespective of who is using this to do what.